esp8266-fastled-webserver/include/configs/controller/controller_esp32.h

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
SIMPLE FILE REVIEW

This header defines default pins to be used when compiling
any of the products to be controlled by an ESP32.

This is not currently the default for any of the existing products.